.babelrc

```diff
@@ -12,13 +12,5 @@
       }
     ]
   ],
-  "plugins": [
-    [
-      "module:@import-meta-env/babel",
-      {
-        "example": "import.meta.env.VERSION"
-      }
-    ]
-  ],
   "comments": false
 }
```

Comment:
newline

Could you please create tasks for "codestyle" ? Need to create editorconfig and maybe renew linter/formatter.

Reply:
Fixed. task created